src/services/change-requests.hooks.ts

```diff
@@ -49,3 +55,23 @@ export const useReviewChangeRequest = () => {
     }
   );
 };
+
+
+/**
+ * Custom React Hook to create a change request.
+ */
+ export const useCreateChangeRequest = () => {
+  return useMutation<{ message: string }, Error, NewChangeRequestPayload>(
+    ['reviewCR'],
```

Comment:
this string is what react-query uses to know when to refetch data I believe, so it needs to be unique or intentionally the same